### PR TITLE
Add WMS / KML separated test

### PR DESCRIPTION
**Pull request by ponceta**
Add WMS layer to the browserstack print test.

Improves testing in the #1991 way

### CODE REVIEW

**Comment by gjn** [comment]
I like the idea! But those are some ugly selectors....are there no other ways to select them (id or class?)

Or: Why not put it directly in a) separate test and b) put the wms directly in the link? (I've proposed in some PR on how to possibly check for the alert message.)

**Comment by gjn** [comment]
Another thing to consider. Now if the print fails, the reasons might be many-fold: print itself, kml loading, wms loading, etc. I also tend to think the standard print (without wms or kml imports) are more important than kml/wms prints.

Maybe a little separation of concerns would make sense. Just thoughts here.

**Comment by procrastinatio** [comment]
I aggree with @gjn, do not over engineer the print test. The result is also visual, and unless you also test the resulting PDF, the test is not complete.

**Comment by ponceta** [comment]
jenkins test

File: test/selenium/kml_test.js [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+  driver.findElement(webdriver.By.xpath("//a[contains(text(), 'Grundlagen und Planung')]"));
+
+  // Click on "Werkzeuge"
+  driver.findElement(webdriver.By.xpath("//a[@id='toolsHeading']")).click();
+  // Click on "KML Import"
+  driver.findElement(webdriver.By.xpath("//*[contains(text(), 'KML Import')]")).click();
+  // Click on "URL"
+  driver.findElement(webdriver.By.xpath("//a[contains(text(), 'URL')]")).click();
+  // Write URL of the chosen KML
+  driver.findElement(webdriver.By.xpath("//*[@id='import-kml-popup']/div[2]/div/div/div[1]/div/div[2]/form/input[1]")).sendKeys('http://opendata.utou.ch/urbanproto/geneva/geo/kml/Routes.kml');
```
Comment (ponceta) [inline test/selenium/kml_test.js]: look for better selector

**Comment by ponceta** [comment]
jenkins test

**Comment by ponceta** [comment]
jenkins test

**Comment by ponceta** [comment]
Please review ;)

File: test/selenium/wms_test.js [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+  // Write URL of the chosen WMS
+  driver.findElement(webdriver.By.xpath("//*[@id='import-wms-popup']//input[@placeholder='URL']")).sendKeys('http://wms.geo.admin.ch/');
+  // Click on "Verbinden"
+  driver.findElement(webdriver.By.xpath("//*[@id='import-wms-popup']//button[contains(text(),'Verbinden')]")).click();
+  // Click on "AGNES"
+  driver.findElement(webdriver.By.xpath("//*[@id='import-wms-popup']//div[contains(text(),'AGNES')]")).click();
+  // Click on "Layer hinzufügen"
+  driver.findElement(webdriver.By.xpath("//*[@id='import-wms-popup']//button[contains(text(),'Layer hinzufügen')]")).click();
+  // Accept the alert
+  driver.switchTo().alert().accept();
```
Comment (gjn) [inline test/selenium/wms_test.js]: If we wait for https://github.com/geoadmin/mf-geoadmin3/pull/2282, we could remove this, right?

**Comment by gjn** [comment]
I like this alot! Good work.

It needs rebase, and maybe we should also wait for https://github.com/geoadmin/mf-geoadmin3/pull/2282

File: test/selenium/wms_test.js (hunk 2 above)
Comment (gjn) [inline test/selenium/wms_test.js]: Maybe not...

Comment (ponceta) [inline test/selenium/wms_test.js]: The alert is the way we assess the user that the layer is now loaded. I do not like alerts but this won't disapears with 3rd party data agreement.

**Comment by ponceta** [comment]
:) This is just separation of concerns for testing, it will not be influenced a lot by #2282.

After #2282 I'll add tests with kml/wms permalinks.

I'll rebase it.

**Comment by gjn** [comment]
Very good improvement. Thanks alot.

**Pull request merged**